src/app/forms/validators/custom-validators.ts

```diff
@@ -180,7 +180,10 @@ export class CustomValidators {
   }
 
   static email(): ValidatorFn {
-    return this.customPattern(['^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$', 'is an invalid format. An email must include an @ symbol and a .']);
+    return this.customPattern([
+      '^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$',
+      'is an invalid format. An email must include an @ symbol and a . and end with two or more characters'
```

Comment:
```suggestion
      'is an invalid format. An email must include an @ symbol, include a full stop and end with two or more characters.'
```
(mostly there are two "and"s in the sentence)

Reply:
the error message on develop does kinda look like a sentence that's not finished